### ISSUE TITLE
Query Filter: Single option filters don't have accessible names

### ISSUE THREAD

**Issue by ryelle** [WordPress/wporg-mu-plugins#645]
Initially reported on learn, this is an issue across the site.

> The toggle that opens the course status filters is labeled using the currently selected status. Buttons to perform an action should be labeled using information about what the control does. Having the current status visible is OK; but it shouldn't be the only content.
>
> This would be better labeled using something like "Filter Courses: All" or even "Filter: All". This would inform the user what they can do with the control.

The single-select variation of query filter should display "[Filter name]: [Value]" as the button text (`label` value).

This has been fixed on Learn in https://github.com/WordPress/Learn/pull/2804

Still needs to be fixed on:

- [x] Themes (only uses multiple select filters)
- [x] Plugins (only has filters if `?show_filters=1`, and this is already using the right format)
- [x] Patterns (curation & sort, remove unused status from "my patterns")
- [x] Showcase (has sort on archive view)

Question for @WordPress/meta-design:

Once we do this, the label is shown twice. Should the label inside the dropdown be hidden again? This would match the multiple select behavior (it's there, but only visible on mobile).

| | Desktop | Mobile |
|---|---|---|
| Multiple select | <img width="426" alt="Screenshot 2024-07-31 at 2 44 08 PM" src="https://github.com/user-attachments/assets/7f057814-79ab-4115-9bdf-5b32c1473132"> | <img width="393" alt="Screenshot 2024-07-31 at 2 43 58 PM" src="https://github.com/user-attachments/assets/209cf677-39f5-4315-a39b-0a39e60da77b"> |
| Single select | <img width="468" alt="Screenshot 2024-07-31 at 2 42 54 PM" src="https://github.com/user-attachments/assets/2d8fd2ab-9b21-4167-a4d0-a49f86e95c3f"> | <img width="392" alt="Screenshot 2024-07-31 at 2 43 15 PM" src="https://github.com/user-attachments/assets/9cc4cf04-d30c-49c0-8002-01178c276852"> |

**Comment by jasmussen** [comment]
> Once we do this, the label is shown twice. Should the label inside the dropdown be hidden again? This would match the multiple select behavior (it's there, but only visible on mobile).

I think we should, yes, but happy to defer to another opinion.

**Comment by ndiego** [comment]
> > Once we do this, the label is shown twice. Should the label inside the dropdown be hidden again? This would match the multiple select behavior (it's there, but only visible on mobile).
> 
> I think we should, yes, but happy to defer to another opinion.

I agree.

**Comment by ryelle** [comment]
Okay, that should be as easy as removing the `&:where(:not(.is-single-select))` from this CSS: https://github.com/WordPress/wporg-mu-plugins/blob/f13cb5f89efd1d6eedbf40624f77c4f42cda1be8/mu-plugins/blocks/query-filter/src/style.pcss#L115-L117

**Comment by fcoveram** [comment]
Agree with removing the label from the dropdown

**Comment by fcoveram** [comment]
I forgot to ask. Does this change include the multi-selection version? The report doesn't mention it.

**Comment by ryelle** [comment]
Multi-selects already use a static label (with dynamic count), ex "Layout [2]", and no title in the desktop dropdown. This change basically makes the two variations more similar.

<img width="447" alt="Screenshot 2024-08-06 at 1 20 03 PM" src="https://github.com/user-attachments/assets/bd5a302a-2143-4df8-9918-1ba46205467c">

**Comment by fcoveram** [comment]
I see. Thanks

**Comment by ryelle** [comment]
The single-select view & labels across the sites have been updated 👍🏻
